database_functions.php

```diff
@@ -94,15 +72,79 @@ function getCategoriesFromDatabase()
             $categories[] = $row;
         }
 
-        // Close the database connection
-        // $conn->close();
-
         return $categories;
     } else {
         // If no categories are found, return an empty array
-        // $conn->close();
         return array();
     }
 }
 
+function getAuctionsFromDatabaseWithParameters($order_by, $category_id, $keyword, $page_num, $page_size) {
```

Comment:
@jasmehr have you handled to show information to user upon this func returning no record?

Reply:
I actually didn't write this code, Anshuman did in his PR and I picked it from there, but I can add code to amend it! @anshuman73 pls cross check if you're satisfied!

Reply:
Ok. So I think this will return empty array to the FE if no result from db, have you handled such conditionals for empty array on the FE side?

If already then g2g. My bad, I should've commented on the my listings php tbh